literature/li_2000_chronocoulometric_95/li_2000_chronocoulometric_95_p1_solid.yaml

```diff
@@ -0,0 +1,105 @@
+          model:
+          supplier:
+        - type:
+          model:
+          supplier:
+    electrolyte:
+        type: aq # aqueous = aq, non-aqueous=naq
+        components: # create as many components as necessary. A simple electrolyte consists of two components, i.e., H2SO4 and water
+            - name: HClO4 # can be trivia name, sum formula, etc
+              concentration:
+                  value: 1
+                  unit: mM #[M, mM, µM, g kg-1, ...]
+              type: #
+              source:
+                  supplier:
+                  LOT: 
+            - name: water
+              type: solvent
+              proportion:
+                  value:
+                  unit:
+              source:
+                  refinement:
+                  quality:
+                  total organic carbon: 
+                      value:
+                      unit:
+            - name:  # can be trivia name, sum formula, etc
+              concentration:
+                  value: 0.1
+                  unit: M #[M, mM, µM, g kg-1, ...]
+              type: #
+              source:
+                  supplier:
+                  LOT: 
+            - name: KClO4
+              type: 
+              source:
+                  supplier:
+                  LOT: 
+              concentration:
+                  value: 0.1
+                  unit: M #[M, mM, µM, g kg-1, ...] 
+        ph: 
+            value: 3
+            uncertainty:
+        temperature: # provide the temperature as it is given in the article. Assume room temperature if nothing is provided
+            value: 298.15 
+            unit: K # [K, C, F]
+    electrodes:
+        configuration: 3 # number of electrodes
+        reference electrode:
+            type: SCE # the reference electrode given in the experimental section. Might differ from that in the plot, MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier:
+        counter electrode:
+            material: Pt
+            crystallographic orientation:
+            shape: coil
+        working electrode:
+            material: Pt
+            crystallographic orientation: '111' # as STRING hkl, 100, 110, poly
+            source:
+                supplier:
+                LOT: 
+            shape: # optional
+                type: # bead/sphere
+                height:
+                    value:
+                    unit:
+                diameter:
+                    value:
+                    unit:
+            geometric electrolyte contact area: 
+                value:
+                unit:
+            preparation procedure: A Pt single crystal bead was cut along the (111) orientation (geometric area of the (111) sur- face 0.043 cm 2 ). The electrode was annealed in a hydro- gen +air flame and then cooled down in a hydrogen atmosphere and later quenched with ultrapure water, before each experiment # Short description of sample preparation. This can be very useful. :)

```

Comment:
```suggestion
            preparation procedure: A Pt single crystal bead was cut along the (111) orientation (geometric area of the (111) surface 0.043 cm2 ). The electrode was annealed in a hydrogen+air flame and then cooled down in a hydrogen atmosphere and later quenched with ultrapure water, before each experiment # Short description of sample preparation. This can be very useful. :)

```